libs/frontend/feature-render-builder/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -50,24 +63,8 @@ const renderImageHover = (
     top = imgComputedStyle.top
     right = imgComputedStyle.right
 
-    if (position === 'absolute' && buildContentWindowInner) {
```

Comment:
Removing this probably broke something, but it seemed to fix selecting/hovering an absolute positioned image. One remaining issue is hovering over the edge of an absolute image/svg causes it to change position in some situations.